android/filament-android/src/main/java/com/google/android/filament/Skybox.java

```diff
@@ -21,6 +21,31 @@
 
 import com.google.android.filament.proguard.UsedByReflection;
 
+/**
+ * Skybox
+ * <p></p>
+ * When added to a Scene, the Skybox fills all untouched pixels.
+ * <p></p>
+ * <h1><u>Creation and destruction</u></h1>
```

Comment:
I don't think we should explicitly underline this header, instead we should allow the CSS of the doc tool to do whatever it wants for h1 elements.